main.c

```diff
@@ -237,20 +239,10 @@ int main(int argc, char **argv)
                 } break;
 
                 case SDLK_F5: {
-                    displayed_time = 0.0f;
-                    paused = false;
```

Comment:
I removed this `paused = false` which was causing the paused state not to be maintained on F5. (Timers should automatically pause when they are restarted, this did the opposite)